repos/spack_repo/builtin/packages/libfabric/package.py

```diff
@@ -170,6 +180,9 @@ class Libfabric(AutotoolsPackage, CudaPackage, ROCmPackage):
         "set by OPX, resulting in undefined behavior.",
     )
 
+    conflicts("+asan", when="+tsan")
+    conflicts("+tsan", when="+asan")
```

Comment:
This can also be expressed more simply as
```suggestion
    conflicts("+asan +tsan")
```
if you like.

Reply:
Oh, much cleaner, thanks! I'll update the PR